### PR TITLE
Merge 1.0-1 to master

### PR DESCRIPTION
**Pull request by secastillo**
*Description of changes:*
Upgrading SKLearn to 1.0-1

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by awsbmillare** [comment]
I see conflicts, can those be resolved or is this intended?

File: ci/buildspec.yml [hunk 1]
```diff
@@ -2,17 +2,26 @@ version: 0.2
...
+      python: 3.9
   pre_build:
     commands:
     - echo Logging in to Amazon ECR...
     - $(aws ecr get-login --no-include-email --region $AWS_DEFAULT_REGION)
+    - echo Installing dependencies...
+    - curl -LO http://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+    - bash Miniconda3-latest-Linux-x86_64.sh -bfp /miniconda3
+    - export PATH=/miniconda3/bin:${PATH}
+    - conda install python=3.7
```
Comment (dewan-c) [inline ci/buildspec.yml:20]: Do we want to use 3.9 here too?

**Comment by dewan-c** [comment]
I'm still not very familiar with sklearn(good thing you also have Nikhil as reviewer :) ) . I don't see a codebuild run automatically linked here. Can you please confirm that there was a successful codebuild run for this PR, or you've run the unit/integ tests locally? Thanks!

File: docker/1.0-1/final/Dockerfile_aarm64.cpu [hunk 2]
```diff
@@ -0,0 +1,56 @@
+FROM sklearn-base:1.0-1-arm64-cpu-py3
```
Comment (dewan-c) [inline docker/1.0-1/final/Dockerfile_aarm64.cpu:1]: This is a separate feature from the sklearn version update right? Not fully sure about the usage of this, but will this be able to find the base image(and this final one) with the specified tag? The base docker file is suffixed as `aarm64` and not `arm64`.

Comment (NikhilRaverkar) [reply docker/1.0-1/final/Dockerfile_aarm64.cpu:1]: The `arm64` is related to graviton image, we can ignore it for now.

Comment (dewan-c) [reply docker/1.0-1/final/Dockerfile_aarm64.cpu:1]: I see, ok.

**Review by NikhilRaverkar — APPROVED** [review]
LGTM

**Review by dewan-c — APPROVED** [review]

**Pull request merged**